scripts/dappnode_install_pre.sh

```diff
@@ -105,6 +105,63 @@ install_iptables () {
     fi
 }
 
+# UNATTENDED UPGRADES INSTALLATION: for upgrading system automatically
+install_unattendedupgrades() {
+    apt-get update -y
+    apt-get install unattended-upgrades -y | tee -a $LOG_FILE
+    if  unattended-upgrades -h >/dev/null 2>&1 ; then
+        echo -e "\e[32m \n\n Verified unattended-upgrades installation \n\n \e[0m" 2>&1 | tee -a $LOG_FILE
+    else
+        echo -e "\e[31m \n\n WARNING: unattended-upgrades not installed, upgrades must be done manually! \n\n \e[0m" 2>&1 | tee -a $LOG_FILE
+    fi
+}
+
+# UNATTENDED UPGRADES SETUP: Enable unattended upgrades and set the configuration
+setup_unattendedupgrades() {
+    # Check and configure unattended-upgrades config file
+    unattended_config_file="/etc/apt/apt.conf.d/50unattended-upgrades"
+    if [ ! -f "$unattended_config_file" ]; then
+        echo -e "\e[31m \n\n WARNING: $unattended_config_file should have been created by the unattended-upgrades package \n\n \e[0m" 2>&1 | tee -a $LOG_FILE
+        return 1
+    fi
+
+    # Enable automatic removal of unused dependencies and disable automatic reboot
+    modify_config_file "$unattended_config_file" 'Unattended-Upgrade::Remove-Unused-Dependencies' 'true'
+    modify_config_file "$unattended_config_file" 'Unattended-Upgrade::Automatic-Reboot' 'false'
+
+    # Check and configure auto-upgrades config file
+    auto_upgrades_file="/etc/apt/apt.conf.d/20auto-upgrades"
+    if [ ! -f "$auto_upgrades_file" ]; then
```

Comment:
regarding upgrades, the official docs encourages also to install the following pkg `apt-listchanges` why is it not been installed?

Check out https://wiki.debian.org/UnattendedUpgrades